.github/workflows/continuous-integration.yml

```diff
@@ -107,14 +107,12 @@ jobs:
       uses: ./.github/actions/setup-libcgroup
     - name: Run functional tests
       run: |
+        # The cgroup v1 runner hosted by Github Actions doesn't allow
+        # for exclusive cpusets.  Thus, skip the cpuset automated test
         pushd tests/ftests
-        make check
+        ./ftests.py -l 10 -L ftests.log
+        ./ftests.py -l 10 -L ftests-nocontainer.log --skip 37 --no-container
```

Comment:
`make check` would also skip testcase `28`. Should it be skipped here too for no-container test?

Reply:
We should probably update `make check`.  Test 028 no longer exists.

Reply:
Agreed, I created a[ PR ](https://github.com/libcgroup/libcgroup-tests/pull/33) to remove the `skip` argument.